07-routing-gossip.md

```diff
@@ -781,23 +781,21 @@ The receiver of `query_channel_range`:
     - MUST set with `chain_hash` equal to that of `query_channel_range`,
     - MUST limit `number_of_blocks` to the maximum number of blocks whose
       results could fit in `encoded_short_ids`
-    - if does not maintain up-to-date channel information for `chain_hash`:
-      - MUST set `full_information` to 0.
-    - otherwise:
-      - SHOULD set `full_information` to 1.
+    - MAY split block contents across multiple `reply_channel_range`.
     - the first `reply_channel_range` message:
 	  - MUST set `first_blocknum` less than or equal to the `first_blocknum` in `query_channel_range`
 	  - MUST set `first_blocknum` plus `number_of_blocks` greater than `first_blocknum` in `query_channel_range`.
 	- successive `reply_channel_range` message:
-	  - MUST set `first_blocknum` to the previous `first_blocknum` plus `number_of_blocks`.
+	  - MUST have `first_blocknum` equal or greater than the previous `first_blocknum`.
+    - MUST set `sync_complete` to `false` if this is not the final `reply_channel_range`.
 	- the final `reply_channel_range` message:
 	  - MUST have `first_blocknum` plus `number_of_blocks` equal or greater than the `query_channel_range` `first_blocknum` plus `number_of_blocks`.
+    - MUST set `sync_complete` to `true`.
```

Comment:
what is `true` in a `byte` typed value? `0x01`?

Reply:
Probably worth opening a PR to fix it, but, yes.

Reply:
Of course, 0x01. Feel free to clarify if you feel it's needed.